<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>824</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>148</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="filename_label">
          <property name="text">
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filename_line_edit">
          <property name="text">
           <string>data/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="seconds_radio_button">
          <property name="text">
           <string>seconds</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">time_units_button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="minutes_radio_button">
          <property name="text">
           <string>minutes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">time_units_button_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="csv_check_box">
          <property name="text">
           <string>.csv</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tsv_check_box">
          <property name="text">
           <string>.tsv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="h5_check_box">
          <property name="text">
           <string>.h5</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>127</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="time_internal_label">
          <property name="text">
           <string>Time interval</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="time_interval_line_edit">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="half_mA_radio_button">
          <property name="text">
           <string>0.5 mA</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">current_button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="five_mA_radio_button">
          <property name="text">
           <string>5 mA</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">current_button_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Control</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="local_radio_button">
          <property name="text">
           <string>Local</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">control_button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="remote_radio_button">
          <property name="text">
           <string>Remote</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">control_button_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Filament</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="filament1_radio_button">
          <property name="text">
           <string>1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filament_button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="filament2_radio_button">
          <property name="text">
           <string>2</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filament_button_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="4">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="gauge_on_button">
          <property name="text">
           <string>Gauge ON</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gauge_off_button">
          <property name="text">
           <string>Gauge OFF</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="com_port_box"/>
        </item>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnect_button">
          <property name="text">
           <string>Disconnect</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_button">
          <property name="text">
           <string>Refresh</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>394</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="6">
     <widget class="MplWidget" name="plot_widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Plotted gauge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ion_gauge_radio_button">
        <property name="text">
         <string>Ion Gauge</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">plotted_gauge_button_gorup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pirani_1_radio_button">
        <property name="text">
         <string>Pirani 1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">plotted_gauge_button_gorup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pirani_2_radio_button">
        <property name="text">
         <string>Pirani 2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">plotted_gauge_button_gorup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="active_gauge_radio_button">
        <property name="text">
         <string>Active Gauge</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">plotted_gauge_button_gorup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="temperature_radio_button">
        <property name="text">
         <string>Temperature</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">plotted_gauge_button_gorup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="next_button">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_save_button">
        <property name="text">
         <string>Start/Save</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="current_button_group"/>
  <buttongroup name="time_units_button_group"/>
  <buttongroup name="single_series_button_group"/>
  <buttongroup name="filament_button_group"/>
  <buttongroup name="control_button_group"/>
  <buttongroup name="plotted_gauge_button_gorup"/>
 </buttongroups>
</ui>
